src/btreemap/node/v1.rs

```diff
@@ -70,15 +70,13 @@ impl<K: Storable + Ord + Clone> Node<K> {
         let mut entries = Vec::with_capacity(header.num_entries as usize);
         let mut offset = NodeHeader::size();
         for _ in 0..header.num_entries {
-            let key_offset = offset;
-            offset += U32_SIZE;
-            let key = LazyKey::by_ref(key_offset);
-            offset += Bytes::from(max_key_size);
+            let size = read_u32(memory, address + offset);
```

Comment:
V1 performance is expected to slowdown a bit, because now it starts reading the size on every load. 
But V1 is deprecated and is only expected to work correctly, users should migrate to V2.
Therefore its performance is not tracked.